internal/cli/opsmanager/clusters/describe.go

```diff
@@ -55,6 +56,14 @@ func (opts *DescribeOpts) Run() error {
 
 	return opts.Print(r)
 }
+func (opts *DescribeOpts) validateArg() error {
```

Comment:
this got me a bit confused, when we list clusters, we get ID and name

```shell
~ ❯ atlas clusters list                                             7s 13:50:52
ID                         NAME           MDB VER   STATE
644a6faa12c63d0ea45ef37a   Cluster99845   6.0.5     CREATING
```

Are we supposed to validate the name as ID? maybe it should be another flag?

Reply:
sorry wait I used altas here let me check OM

Reply:
>maybe it should be another flag?
 
```
mongocli ops-manager clusters describe --help
When describing cluster with no output format please provide the cluster ID.
When using an output format the please provide the cluster name.
```


At this point any change would be a breaking change there's a ticket to figure out what to do with these commands but it hasn't been prioritized so far

Reply:
ah, got it! thanks for the clarification